polln_shift/models/planning.py

```diff
@@ -41,6 +41,39 @@ def subscribe(self):
         return res
 
 
+class TaskTemplate(models.Model):
+    _inherit = "beesdoo.shift.template"
+
+    def name_get(self):
+        res = []
+        for rec in self:
```

Comment:
We try to use more explicit names in our loops, such as `for template in self:` or `for shift_template in self:`. Don't you use that at Odoo ?

Reply:
> Don't you use that at Odoo ?
There is no specific guidelines for that in odoo (you can grep for rec in self, for record in self) 
Personnaly when the record come from a search it makes sense to specify the object
when they come from self, you already know in which model you are, it's less important